test/periphery/integration/merkle-campaign/instant/claim/claim.t.sol

```diff
@@ -1,106 +1,18 @@
-        _;
-    }
-
-    modifier givenNotIncludedInMerkleTree() {
-        _;
-    }
-
-    function test_RevertWhen_InvalidIndex()
-        external
-        givenCampaignNotExpired
-        givenNotClaimed
-        givenNotIncludedInMerkleTree
-    {
-        uint256 invalidIndex = 1337;
-        uint128 amount = defaults.CLAIM_AMOUNT();
-        bytes32[] memory merkleProof = defaults.index1Proof();
-        vm.expectRevert(abi.encodeWithSelector(Errors.SablierMerkleBase_InvalidProof.selector));
-        merkleInstant.claim(invalidIndex, users.recipient1, amount, merkleProof);
-    }
-
-    function test_RevertWhen_InvalidRecipient()
-        external
-        givenCampaignNotExpired
-        givenNotClaimed
-        givenNotIncludedInMerkleTree
-    {
-        uint256 index1 = defaults.INDEX1();
-        address invalidRecipient = address(1337);
-        uint128 amount = defaults.CLAIM_AMOUNT();
-        bytes32[] memory merkleProof = defaults.index1Proof();
-        vm.expectRevert(abi.encodeWithSelector(Errors.SablierMerkleBase_InvalidProof.selector));
-        merkleInstant.claim(index1, invalidRecipient, amount, merkleProof);
-    }
-
-    function test_RevertWhen_InvalidAmount()
-        external
-        givenCampaignNotExpired
-        givenNotClaimed
-        givenNotIncludedInMerkleTree
-    {
-        uint256 index1 = defaults.INDEX1();
-        uint128 invalidAmount = 1337;
-        bytes32[] memory merkleProof = defaults.index1Proof();
-        vm.expectRevert(abi.encodeWithSelector(Errors.SablierMerkleBase_InvalidProof.selector));
-        merkleInstant.claim(index1, users.recipient1, invalidAmount, merkleProof);
-    }
-
-    function test_RevertWhen_InvalidMerkleProof()
-        external
-        givenCampaignNotExpired
-        givenNotClaimed
-        givenNotIncludedInMerkleTree
-    {
-        uint256 index1 = defaults.INDEX1();
-        uint128 amount = defaults.CLAIM_AMOUNT();
-        bytes32[] memory invalidMerkleProof = defaults.index2Proof();
-        vm.expectRevert(abi.encodeWithSelector(Errors.SablierMerkleBase_InvalidProof.selector));
-        merkleInstant.claim(index1, users.recipient1, amount, invalidMerkleProof);
-    }
-
-    modifier givenIncludedInMerkleTree() {
-        _;
+contract Claim_MerkleInstant_Integration_Test is MerkleCampaign_Integration_Shared_Test {
+    function setUp() public override {
+        super.setUp();
+        merkleBase = merkleInstant;
     }
 
     function test_Claim() external givenCampaignNotExpired givenNotClaimed givenIncludedInMerkleTree {
-        vm.expectEmit({ emitter: address(merkleInstant) });
+        vm.expectEmit({ emitter: address(merkleBase) });
         emit Claim(defaults.INDEX1(), users.recipient1, defaults.CLAIM_AMOUNT());
-
         expectCallToTransfer({ to: users.recipient1, value: defaults.CLAIM_AMOUNT() });
-
-        claimInstant();
-
-        assertTrue(merkleInstant.hasClaimed(defaults.INDEX1()), "not claimed");
+        claim();
```

Comment:
Some newlines between these would be nice especially after emit statement.

Reply:
so, this is your suggestion?

```solidity
vm.expectEmit({ emitter: address(merkleBase) });
emit Claim(defaults.INDEX1(), users.recipient1, defaults.CLAIM_AMOUNT());

expectCallToTransfer({ to: users.recipient1, value: defaults.CLAIM_AMOUNT() });
claim();
```

Reply:
Yes.